RELEASE/scripts/sl_ascend.ash

```diff
@@ -4762,6 +4762,12 @@ boolean L13_towerNSContests()
 			stat crowd_stat = ns_crowd2();
 			if (my_path() == "Dark Gyffte")
 			{
+				if(crowd_stat == $stat[muscle] && !have_skill($skill[Preternatural Strength]))
+				{
+					boolean[skill] requirements;
+					requirements[$skill[Preternatural Strength]] = true;
+					bat_pickSkills(20, requirements);
```

Comment:
sorry for the confusing comment, I should have gotten rid of `// to be called when already in torpor`. bat_pickSkills only gets a list of skills to take, it doesn't actually perform any action. So you'd need to call bat_reallyPickSkills here, and refactor IT to take requirements.

Reply:
Fixed and renamed for clarity